### PR TITLE
LRIS red mark4 mask ingestion

### PR DESCRIPTION
**Pull request by SunilSimha**
As titled. I've tweaked the code and now it correctly ingests mask design information for the red side mark4 detector.

### CODE REVIEW

File: pypeit/spectrographs/keck_lris.py [hunk 1]
```diff
@@ -583,7 +583,7 @@ def get_maskdef_slitedges(self, ccdnum=None, filename=None, debug=None,
             sep = mask_coord.separation(slit_coords[islit])
             PA = mask_coord.position_angle(slit_coords[islit])
             #
-            alpha = sep.to('arcsec') * np.cos(PA-self.slitmask.posx_pa*units.deg)
+            alpha = sep.to('arcsec') * np.cos(PA-self.slitmask.posx_pa*units.deg-180*units.deg)
```
Comment (profxj) [inline pypeit/spectrographs/keck_lris.py]: is this `-180deg` rotation true for *all* LRIS detectors, or only `mark4`?

**Review by profxj — CHANGES_REQUESTED** [review]
Thanks for this.

Please make sure to add a test in the DevSuite
which includes a `vet` test that insures things 
are behaving as desired.

I can help

File: pypeit/spectrographs/keck_lris.py (hunk 1 above)
Comment (SunilSimha) [reply pypeit/spectrographs/keck_lris.py]: Oops. That's not supposed to be there at all. :)
Removed.

**Comment by SunilSimha** [comment]
Happy to include the test in the dev suite. Although, I'll need edit permissions to add files in that folder.

File: pypeit/spectrographs/keck_lris.py (hunk 1 above)
Comment (profxj) [reply pypeit/spectrographs/keck_lris.py]: but how was it working with it?

**Review by debora-pe — CHANGES_REQUESTED** [review]
Hey @SunilSimha I'm OK with these changes here. As we discussed we should add some tests. Let me know if you need any help.

**Review by profxj — APPROVED** [review]
Approving

**Review by debora-pe — APPROVED** [review]
Approving

**Pull request merged**